variables.tf

```diff
@@ -372,7 +372,7 @@ variable "subnets" {
   description = "Subnets to be created and their settings - This variable is ignored when `subnet_ids` is set (AKA bring your own subnets)."
   type        = map(list(string))
   default = {
-    "private" : ["192.168.0.0/18"], # multi-zonal cluster is created by adding additional subnets here
+    "private" : ["192.168.0.0/18"],                              # multi-zonal cluster is created by adding additional subnets here
```

Comment:
All the changes look good, may need a rebase the comment was removed in #241

Reply:
Thanks @jarpat, it looks like that same comment was removed from CONFIG-VARS.md but remained in variables.tf, I'll remove it here as well.